examples/with-typescript-recommended/src/listeners/commands/contextMenuCommands/contextMenuCommandSuccessLogger.ts

```diff
@@ -0,0 +1,41 @@
+import { ApplyOptions } from '@sapphire/decorators';
```

Comment:
Look at https://github.com/favware/archangel/blob/main/src/listeners/commands/chatInputCommands/chatInputCommandSuccessLoger.ts to get an idea of how to ensure there is no duplicate code between context menu and chat input